### ISSUE TITLE
Declaration Generator/LG: Shift line down to closing paragraph

### ISSUE THREAD

**Issue by SamHyler** [hackforla/expunge-assist#1503]
### Title

Shift the final line down to the closing paragraph

### Overview

High priority quick fix issue from Bonnie's feedback (https://github.com/hackforla/expunge-assist/issues/1461).
The final line in the final paragraph ("Why") before the closing in the LG/Declaration generator needs to be moved to the closing paragraph.   

### Action Items

- [x] Check off these boxes as they are done and document progress and updates in the comments below
- [x] Find the final two paragraphs ("why" and the closing paragraph) in Figma
- [x] Move the final line that is pre-written by EA/the generator in the "Why" paragraph to the closing paragraph. The line says "Thank you for considering my case."
- [x] Update final copy in all 3 spaces: Mobile, Desktop, and the Content SOT on Google Docs
- [ ] Tag the Content leads on this issue by adding the label ready for content lead and pinging on slack
- [x] Create issue for Dev 
- [x] handoff to Dev [#1681 ](https://github.com/hackforla/expunge-assist/issues/1681)
   - [x] add deliverable(design or content)
   - [x] remove https://github.com/hackforla/expunge-assist/labels/dependency label
   - [x] add https://github.com/hackforla/expunge-assist/labels/ready%20for%20dev%20lead label
   - [x] move to `new issue approval` column

### Resources/Notes

Work was done directly in the master pages, here is the link to the Design-Dev hand off space https://www.figma.com/design/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?node-id=18520-35533&m=dev

**Comment by cjankowski23** [comment]
Found the final two paragraphs ("why" and the closing paragraph) in Figma:

Locations:
[Advice Page + Input Field](https://www.figma.com/design/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?node-id=2-6)
[Master Pages: Desktop](https://www.figma.com/design/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?node-id=2-35)
[Master Pages: Mobile](https://www.figma.com/design/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?node-id=2-36)

Shifted final line in all three locations.

Updated SOT on Google Docs
[Expunge Assist SOT](https://docs.google.com/spreadsheets/d/1c_Kv_T1yER-hHckNl5zkGrlGaLNCE_AeZE6Jjmn8IK0/edit?gid=522832564#gid=522832564)

**Comment by cjankowski23** [comment]
**Progress:** 
Noticed when checking the Figma files sometimes the "Thank you for your consideration" statement was at the beginning of the why paragraphs, other times at the end of the why section. I audited the live form and messaged @SamHyler for clarification. 

The line has now been shifted into its own closing line separate from the "Why" page.

**Estimated Time to Completion (ETC):** 1 week

**Blockers:**
Messaged @anitadesigns and @jyehllow on slack to check spacing is visually correct on the master draft for mobile and desktop before handing off to Dev.

**Comment by anitadesigns** [comment]
I reviewed 2 frames (1 desktop and its mobile correspondent) and have questions for 2 other frames that I left on Figma! If those two frames add copy changes, I can review the padding once it is confirmed!

**Comment by SamHyler** [comment]
This has been done for a long time but (as I understand it?) didn't get handed over. Waiting on one frame -something happened to the order of the sections within it. Otherwise this is done and Dev can implement. 

https://www.figma.com/design/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?node-id=18520-35533&m=dev

**Comment by SamHyler** [comment]
Confirmed by @jyehllow that Design parts are ready to go. Handing over to Dev. Issue complete.
